roles/provision-unifiedpush-apb/templates/secret.yml.j2

```diff
@@ -7,6 +7,7 @@ metadata:
     name: ups
     mobile: enabled
     serviceName: ups
+    serviceInstanceID: {{ _apb_service_instance_id }}
```

Comment:
@JameelB as @psturc has mentioned, `_apb_service_instance_id` is not available when testing. You could create another template named secret_testing.yml.j2 that does not have the `serviceInstanceID` and then in your ansible role use something like this:

```
- name: "Create ups secret yaml file"
  template:
    src: secret.yml.j2
    dest: /tmp/secret.yaml
  when: _apb_service_instance_id is defined

- name: "Create ups testing secret yaml file"
  template:
    src: secret_testing.yml.j2
    dest: /tmp/secret.yaml
  when: _apb_service_instance_id is not defined
```

Reply:
@pb82 Sure i'll add that change in :+1:

Reply:
ping @maleck13 @dimitraz